### PR TITLE
Issue #DP-602 Fix:Label changes for the statuses shown

### PR DESCRIPTION
**Pull request by Nilesh5995**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts [hunk 1]
```diff
@@ -185,6 +185,7 @@ export class ChapterListComponent implements OnInit, OnChanges, OnDestroy, After
   }
 
   public initiateInputs(action?, content?) {
+    const sourcingStatus = content ? content.sourcingStatus :null;
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts:188]: you will need to check if the content is defined

File: src/app/client/src/app/modules/cbse-program/components/content-uploader/content-uploader.component.html [hunk 2]
```diff
@@ -68,20 +68,20 @@
...
+              <button id="requestChanges" suiPopup popupText="Reject" [popupPlacement]="'top'" *ngIf="visibility && visibility.showRequestChanges && resourceStatus === 'Review'" class="sb-btn sb-btn-error sb-btn-normal mr-8" (click)="showRequestChangesPopup = true;"
               appTelemetryInteract
                 [telemetryInteractEdata]="programTelemetryService.getTelemetryInteractEdata('reject-resource', 'click', 'content-uploader')"
                 [telemetryInteractObject]="telemetryInteractObject"
                 [telemetryInteractCdata]="telemetryInteractCdata"
                 [telemetryInteractPdata]="telemetryInteractPdata"
-              >Request Changes</button>
-              <button id="publishContent" suiPopup popupText="Publish" [popupPlacement]="'top'" *ngIf="visibility && visibility.showPublish && resourceStatus === 'Review'" class="sb-btn sb-btn-secondary sb-btn-normal mr-8" (click)="publishContent();"
+              >Reject</button>
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/content-uploader/content-uploader.component.html:77]: Can this be lang constants?

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts [hunk 3]
```diff
@@ -479,6 +485,28 @@ export class ChapterListComponent implements OnInit, OnChanges, OnDestroy, After
...
+  }
+
+  checkSourcingStatus(content) {
+    if (this.storedCollectionData.acceptedContents  &&
+         _.includes(this.storedCollectionData.acceptedContents || [], content.identifier)) {
+            return 'Approved';
+      } else if (this.storedCollectionData.rejectedContents  &&
+              _.includes(this.storedCollectionData.rejectedContents || [], content.identifier)) {
+                 return 'Rejected';
+      } else {
```
Comment (niteshkesarkar) [inline src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts:505]: @Nilesh5995 - No need of `else` block. just add `return null;`

File: src/app/client/src/app/modules/cbse-program/components/content-uploader/content-uploader.component.ts [hunk 4]
```diff
@@ -72,6 +72,7 @@ export class ContentUploaderComponent implements OnInit, AfterViewInit, OnDestro
   public telemetryInteractObject: any;
   public telemetryPageId = 'content-uploader';
   public sourcingOrgReviewer: boolean;
+  public sourcingReviewStatus: any;
```
Comment (niteshkesarkar) [inline src/app/client/src/app/modules/cbse-program/components/content-uploader/content-uploader.component.ts:75]: @Nilesh5995 `sourcingReviewStatus` should be `string`, right? It should not be any.

File: src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts [hunk 5]
```diff
@@ -71,6 +71,7 @@ export class QuestionListComponent implements OnInit, AfterViewInit, OnDestroy {
   telemetryImpression: any;
   public telemetryPageId = 'question-list';
   public sourcingOrgReviewer: boolean;
+  public sourcingReviewStatus: any;
```
Comment (niteshkesarkar) [inline src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts:74]: @Nilesh5995 sourcingReviewStatus should be string, right? It should not be any.

File: src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts [hunk 6]
```diff
@@ -96,6 +97,7 @@ export class QuestionListComponent implements OnInit, AfterViewInit, OnDestroy {
     this.sessionContext.questionType = this.templateDetails.questionCategories[0];
     this.sessionContext.textBookUnitIdentifier = _.get(this.practiceQuestionSetComponentInput, 'unitIdentifier');
     this.practiceSetConfig = _.get(this.practiceQuestionSetComponentInput, 'config');
+    this.sourcingReviewStatus = _.get(this.practiceQuestionSetComponentInput, 'sourcingStatus') || null;
```
Comment (niteshkesarkar) [inline src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts:100]: @Nilesh5995 if `sourcingReviewStatus` is set to `string`, then default value should be `''`

File: src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts [hunk 7]
```diff
@@ -204,11 +206,15 @@ export class QuestionListComponent implements OnInit, AfterViewInit, OnDestroy {
...
   setResourceStatus() {
     if (this.resourceStatus === 'Review') {
-      this.resourceStatusText = 'Review in Progress';
+      this.resourceStatusText = this.resourceService.frmelmnts.lbl.reviewInProgress;
     } else if (this.resourceStatus === 'Draft' && this.resourceDetails.rejectComment && this.resourceDetails.rejectComment !== '') {
-      this.resourceStatusText = 'Rejected';
-    } else if (this.resourceStatus === 'Live') {
-      this.resourceStatusText = 'Published';
+      this.resourceStatusText = this.resourceService.frmelmnts.lbl.notAccepted;
+    } else if (this.resourceStatus === 'Live' && !this.sourcingReviewStatus) {
```
Comment (niteshkesarkar) [inline src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts:212]: @Nilesh5995 if sourcingReviewStatus is set to string, then add check like `!_.isEmpty(this.sourcingReviewStatus)`

**Pull request merged**